### DISCUSSION TITLE
Stable-fast: a TensorRT alternative

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#2417] [General]
https://www.reddit.com/r/StableDiffusion/comments/179uuz3/introducing_stable_fast_an_ultra_lightweight/

This was probably discussed in the vlad discord before, but I just wanted to make sure this does not fly under the radar. It sounds incredibly good, but I do not know, how much effort it is, to implement this hack (even via model.compile only)

The scientist / researcher seems to be very open about their techniques and wants to communicate. Is this an easy task to integrate? I never noticed a difference between model compile and normal mode in backend ORIGINAL, so I think anyway, this might be a considerable hack to increase it/s

I wish there was some compatibility talk between this repo and this guy

**Comment by vladmandic** [comment]
already on radar, see open projects: https://github.com/users/vladmandic/projects/2?pane=issue&itemId=42265095

**Reply by blacklig** [reply]
How do we use it in Next.SD then? And does it work with original back end or only diffusers? Because in diffusers we dont have CNs so that would be sad if only Diffusers supported...

**Reply by vladmandic** [reply]
Notes are in the changelog.
